tests/data/dsl-semantic-validation-cases.yaml

```diff
@@ -746,8 +746,8 @@
         start: 7
         end: 7
       column:
-        start: 0
-        end: 10
+        start: 1
```

Comment:
This is an example of where we were potentially returning a zero based line number instead of a 1 based line number